src/backend/metal/src/command.rs

```diff
@@ -2200,7 +2200,9 @@ impl com::RawCommandBuffer<Backend> for CommandBuffer {
         }
 
         let sink = inner.sink();
-        sink.blit_commands(blit_commands.into_iter());
+        if !blit_commands.is_empty() {
+            sink.blit_commands(blit_commands.into_iter());
```

Comment:
Should we simply move this check to the beginning of `blit_commands` instead? It seems like there are already some other places where an empty iterator could be passed

Reply:
It doesn't seem convenient, since we have an iterator, so we'd have to mess with some lazy initialization or fixed-length iterator traits, etc. Doing it here is straightforward.